wsmaster/che-core-api-workspace-activity/src/main/java/org/eclipse/che/api/workspace/activity/WorkspaceActivity.java

```diff
@@ -105,6 +106,9 @@
   @Column(name = "expiration")
   private Long expiration;
 
+  @Column(name = "workspace_hard_expiration")
+  private Long workspaceHardExpiration;
```

Comment:
why do we need extra field if we have already  `   last_starting              BIGINT and   last_running               BIGINT`, which can be used for how long workspace is already running?